### PR TITLE
AWS release  9.0.10

### PR DESCRIPTION
**Pull request by calvix**
release to fix bug https://github.com/giantswarm/giantswarm/issues/12450

### CODE REVIEW

**Review by njuettner — APPROVED** [review]
lgtm

File: aws/v9.0.10/release.diff [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 0.23.14                                                   version: 0.23.14
+  - name: containerlinux                                             - name: containerlinux
+    version: 2512.2.1                                                  version: 2512.2.1
+  - name: etcd                                                       - name: etcd
+    version: 3.3.15                                                    version: 3.3.15
+  - name: kubernetes                                                 - name: kubernetes
+    version: 1.15.12                                                   version: 1.15.12
+  date: "2020-07-31T15:00:00Z"                                  |    date: "2020-08-19T15:34:28Z"
```
Comment (njuettner) [inline aws/v9.0.10/release.diff:53]: Date needs to be modified again

File: aws/v9.0.10/release.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 0.23.14
+  - name: containerlinux
+    version: 2512.2.1
+  - name: etcd
+    version: 3.3.15
+  - name: kubernetes
+    version: 1.15.12
+  date: "2020-08-19T15:34:28Z"
```
Comment (njuettner) [inline aws/v9.0.10/release.yaml:53]: also here

**Comment by calvix** [comment]
cluster creation `v9.v.10` and update from `v9.0.9` to `v9.0.10`

**Pull request merged**